meilisearch/src/option.rs

```diff
@@ -287,11 +286,12 @@ pub struct Opt {
     #[serde(default)]
     pub log_level: LogLevel,
 
-    /// Enables Prometheus metrics and /metrics route.
-    #[cfg(feature = "metrics")]
-    #[clap(long, env = MEILI_ENABLE_METRICS_ROUTE)]
+    /// Experimental metrics feature. For more information, see: https://github.com/meilisearch/meilisearch/discussions/3518
```

Comment:
```suggestion
    /// Experimental metrics feature. For more information, see: <https://github.com/meilisearch/meilisearch/discussions/3518>
```

(rustdocs comments are markdown. Stray URLs in markdown need to be enclosed in `<` and `>`)